ProcessAPI/ProcessTest.java

```diff
@@ -0,0 +1,12 @@
+public class ProcessTest {
+
+    public void processTest(ComputeEngine engine)
+    {
+        //check input and output source 
+        if(engine.processData(null, null).isValid()) //(null for now, until Sources are filled in)
+        {
+            engine.compute();

```

Comment:
Very nice prototype! This particular prototype will need a ProcessInterface and a ConceptualAPI - the processData call will be on the process interface, and then compute gets called on the conceptual api.